### ISSUE TITLE
Reproduce ApeX paper results for continuous action

### ISSUE THREAD

**Issue by keiohta** [keiohta/tf2rl#22]
[Distributed Prioritized Experience Replay](https://arxiv.org/abs/1803.00933)
Hyper parameters are shown in appendix D

**Comment by keiohta** [comment]
- Critic: [400, 300]
- Actor: [300, 200]
- Gradient clipping [-1, 1], element-wise
- Adam with learning rate of 0.0001
-  The target network is copied from the online network every
100 training batches
- Priority experience replay
    - alpha_sample = 0.6
    - alpha_evict = -0.4
- Noise is added to DDPG with sigma = 0.3

**Comment by keiohta** [comment]
Waiting for `cpprb` implementation of `NStepReplayBuffer`.
